spec/factories/restaurant_factory.rb

```diff
@@ -49,5 +49,18 @@
       max_delivery_time_min { 720 }
       accepts_10bis { false }
     end
```

Comment:
you need to add an empty line after each block :) 
check please if RoboCup scream on you, and if not, why :)

Reply:
added lineSpaces but didn't find a layout cop for this in the docs,
can you link the .rubocop.yml you guys are using?

Reply:
tried to find in our projects, didn't find a rule for this. but you can read our style-guide: https://github.com/WeConnect/style-guide/tree/master/ruby#newlines (not really up to date, but we follow Airbnb style-guide and this section is the same)